setup/service/dhis2-create-instance

```diff
@@ -148,7 +148,24 @@ then
   lxc exec $NAME -- sed -i "s/^connection.url.*$/connection.url = jdbc:postgresql:\/\/$PG\/$NAME/" /opt/dhis2/dhis.conf
 fi
 
-lxc file push /usr/local/etc/dhis/tomcat_default $NAME/etc/default/tomcat9
+APM=$(cat configs/containers.json| jq -r .apm)
```

Comment:
Same problem as proxy type.  This will only work if run from setup directory.  Maybe solution is to copy containers.json to /usr/local/etc/dhis2

Reply:
I have done two commits:
- Copy container.json to /usr/local/etc/dhis/ in install_scripts.sh and added /usr/local/etc/dhis/ as absolute path in my previous changes
- Add /usr/local/etc/dhis/containers.json as absolute path in munin_node.sh and parse_config.sh